index.html

```diff
@@ -8,16 +8,52 @@
     <script src="app.js" defer></script>
     <title>Awesome Books Website</title>
   </head>
+   <header id="header">
+      <nav class="nav">
+        <a href="#">Awesome Books</a>
+        <!-- <div class="nav-link">
+          <a class="booklist" href="#">List</a>
+          <a class="addition" href="#">Add new</a>
+          <a class="contact" href="#">Contact</a>
+        </div> -->
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻